lib/assent/strategies/oauth2.ex

```diff
@@ -166,7 +166,7 @@ defmodule Assent.Strategy.OAuth2 do
   defp authentication_params(:private_key_jwt, config) do
     alg = Config.get(config, :jwt_algorithm, "RS256")
 
-    with {:ok, pem}             <- load_private_key(config),
+    with {:ok, pem}             <- __load_private_key__(config),
```

Comment:
Could we remove these `__func_name__` functions?

Is there any reason why can't be named without `__`?

Reply:
It's to keep it as a "private" API method. I didn't feel it was a good idea to expose it as public method for OAuth2. Probably makes more sense to move this to a helper module for JWT though.